apps/public-docsite-v9/src/Concepts/Migration/FromV0/Icons.stories.mdx

```diff
@@ -3,7 +3,9 @@ import { IconCatalog } from './Components/IconCatalog/IconCatalog.tsx';
 
 <Meta title="Concepts/Migration/from v0/Icons" />
 
-# v0 - v9 Icon Catalog
+# Icons Migration
+
+## v0 - v9 Icon Catalog
```

Comment:
I would place the icon catalog as the last section

Reply:
Moved https://github.com/microsoft/fluentui/pull/25828/commits/12c0af7e66ee407db13bc9ac99047a2dd266e43c